### ISSUE TITLE
tried to register two views with the same name GestureHandlerRootView

### ISSUE THREAD

**Issue by hackthemachin3** [react-native-community/react-native-navbar#257]
As you can see in the screenshot, I get the issue after i install react-navigation and react-gesture-handler
![screenshot_20190130-014721](https://user-images.githubusercontent.com/46753886/51953394-1d4d1100-2434-11e9-9a11-d226bfe60211.png)

**Comment by AlexisChup** [comment]
I have the same error and I don't find the answer. Furthermore it is the same context, i follow a tuto and after install react-navigation and react-gesture-handler, and try to use a stack navigator, i have this screen on my emulator. Does someone knows why ?
Thanks

**Comment by hackthemachin3** [comment]
Yes I had the same problem it's because you have two version of react native gesture handler u have to delete the one in the folder of your app: node_module/expo/node_modules/react-native-gesture-handler

**Comment by AlexisChup** [comment]
Honestly, THANK YOU, 5 hours that i'm were blocked.
Have a good day !

**Comment by oliv3** [comment]
Had exactly the same problem, @hackthemachin3's fix did the job.

**Comment by georgeprice** [comment]
Same for me, thanks!

**Comment by stevenkeezer** [comment]
hello

**Comment by stevenkeezer** [comment]
test

**Comment by stevenkeezer** [comment]
test

**Comment by stevenkeezer** [comment]
Apologize for all the tests.

**Comment by stevenkeezer** [comment]
test
